src/bokeh/io/notebook.py

```diff
@@ -498,6 +500,41 @@ def publish_display_data(data: dict[str, Any], metadata: dict[Any, Any] | None =
     from IPython.display import publish_display_data
     publish_display_data(data, metadata, transient=transient, **kwargs)
 
+def _remote_jupyter_proxy_url(port: [int | None]) -> str:
```

Comment:
no brackets AFAIK
```suggestion
def _remote_jupyter_proxy_url(port: int | None) -> str:
```
also docstring is out of sync now